### ISSUE TITLE
ML: Gradient boosting assignment

### ISSUE THREAD

**Issue by zhangqi444** [ocademy-ai/machine-learning#223]
assignment:  

- [x] https://www.kaggle.com/code/elyousfiomar/hyperparameter-tuning-gradient-boosting 

-  https://www.kaggle.com/code/shahules/tackling-class-imbalance ：The data set is too large, 600M

- [x] https://www.kaggle.com/code/angqx95/data-science-workflow-top-2-with-tuning  

-  https://www.kaggle.com/code/edankinderman/default-prediction-gradient-boosting ：The data set is too large, several G

- https://www.kaggle.com/code/gpreda/breast-cancer-prediction-from-cytopathology-data ：Using the R language

-  https://www.kaggle.com/code/alexisbcook/exercise-xgboost ：The code is incomplete, the job itself is knocked out

- https://www.kaggle.com/code/abhinand05/catboost-a-deeper-dive ：no dataset

- [ ] https://www.kaggle.com/code/azminetoushikwasi/xgboost-wrangling-with-hyperparameters-guide 

- [ ] https://www.kaggle.com/code/arthurtok/introduction-to-ensembling-stacking-in-python
